tools/istio-iptables/pkg/cmd/run.go

```diff
@@ -145,12 +145,20 @@ func (iptConfigurator *IptablesConfigurator) handleInboundPortsInclude() {
 				//TODO: (abhide): Move this out of this method
 				iptConfigurator.ext.RunOrFail(constants.IP, "route", "show", "table", "all")
 			}
+
+			ifaceName := getNIC()
+			if ifaceName != "" {
+				// set route local net
+				iptConfigurator.ext.RunOrFail(
+					"sysctl", "-w", fmt.Sprintf("net.ipv4.conf.%s.route_localnet=1", ifaceName))
```

Comment:
Will this break distroless?

Reply:
no, proxy image contains `sysctl`

Reply:
that doesn't seem to be true

```
$ docker run -it --entrypoint "" istio/proxyv2:1.8.0-distroless sysctl
docker: Error response from daemon: OCI runtime create failed: container_linux.go:370: starting container process caused: exec: "sysctl": executable file not found in $PATH: unknown.
```

Reply:
Oh proxyv2 is different from the distroless. We must document this.

By the way, for the `enable-core-dump` container, it has the same issue

Reply:
@sdake  @howardjohn How do we build istio/proxyv2:1.8.0-distroless? Is it possible to add `sysctl`? The enable-core-dump feature also needs it.